README.md

```diff
@@ -102,8 +102,3 @@ Here's a table:
 | Empty                 | []                    | {"L": []}                                          |
 | Mixed                 | [42, "foo", null]     | {"L": [{"N": "42"}, {"S": "foo"}, {"NULL": true}]} |
 | Duplicates            | ["foo", "bar", "foo"] | {"L": [{"S": "foo"}, {"S": "bar"}, {"S": "foo"}]}  |
```

Comment:
The style guide part was a  bit outdated too, so I just axed it. You have a fair point though, I'll put something back in which encourages contributions.